CMakeLists.txt

```diff
@@ -317,6 +317,12 @@ set(CMAKE_CXX_FLAGS_COVERAGE "-fprofile-arcs -ftest-coverage")
 include(CheckLibraryExists)
 check_library_exists(m pow "" HAVE_LIBM)
 
+#-----------------------------------------------------------------------------
+# Extra include
+#-----------------------------------------------------------------------------
+include(CheckIncludeFile)
+check_include_file(fenv.h HAVE_FENV_H)
```

Comment:
It might be a bit safer to use [check_include_file_cxx](https://cmake.org/cmake/help/latest/module/CheckIncludeFileCXX.html) (after including `CheckIncludeFileCXX`) to check directly for `cfenv`. Not sure if there's any platforms where the C include is present but C++ include is not, but might provide a bit more piece of mind if it's a trivial change.